### PR TITLE
Fix plugin embed link

### PR DESCRIPTION
**Pull request by asportnoy**
Fix installer/gh links always showing up as just the github link if no data:
`https://replugged.dev/install?url=whatever/whatever` would show as `https://github.com/whatever/whatever`
`[some named link](https://github.com/whatever/whatever)` would show as `https://github.com/whatever/whatever`

### CODE REVIEW

**Comment by asportnoy** [comment]
Need to merge #191 before this bc I based it on the wrong branch

File: src/Powercord/coremods/plugin-embeds/components/PluginEmbed.jsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ const {
   subHead
 } = getModule([ 'titleRegion' ], false);
 
-module.exports = function ({ match }) {
+module.exports = function ({ match, props }) {
```
Comment (lleyton) [inline src/Powercord/coremods/plugin-embeds/components/PluginEmbed.jsx:40]: seems a bit strange to take a nested props object, why don't we just take in an href and title prop?

**Review by lleyton — CHANGES_REQUESTED** [review]

File: src/Powercord/coremods/plugin-embeds/components/PluginEmbed.jsx (hunk 1 above)
Comment (asportnoy) [reply src/Powercord/coremods/plugin-embeds/components/PluginEmbed.jsx:40]: match is also a nested object. I don't personally see an issue with it.

Comment (lleyton) [reply src/Powercord/coremods/plugin-embeds/components/PluginEmbed.jsx:40]: mmm, fair enough

**Review by lleyton — APPROVED** [review]
LGTM, but gonna wait on the cute cat to take a look

**Review by lexisother — APPROVED** [review]

**Comment by lexisother** [comment]
Rebase time :trollface:

**Pull request merged**